.github/workflows/compile-macos-intel.yaml

```
@@ -0,0 +1,35 @@
+name: MacOS(ARM64)
+
+on: [pull_request]
+
+jobs:
+  compile-MacOS-ARM64:
+    runs-on: macos-latest
+    steps:
+    - name: Check Platform
+      run: |
+        if [ ! $(uname -m) = "arm64" ]; then
+            echo "Not on ARM64" >&2
+            exit 1
+        fi
+    - name: Setup MacOS
```

What's in a name...?

```suggestion
    - name: Setup macOS
```

---

Thank you. Fixed.